README.md

```diff
@@ -74,6 +74,62 @@ Attention: `supportContextStrategyChecks` and `checkNameForContextStrategyByToke
  | neverSuppressedChecks                            | String has user defined TreeWalker Checks to never suppress if files are touched, split by comma. This property is useful for Checks that place violation on whole file or on not all (first/last) occurrence of cause/violated code. | [String](https://checkstyle.sourceforge.io/property_types.html#String) | null          |
  | checkNamesForContextStrategyByTokenOrParentSet | String has user defined TreeWalker Checks that need modify violation nodes to their parent node to expand the context scope, split by comma | [String](https://checkstyle.sourceforge.io/property_types.html#String) | null          |
 
+#### Notes
+
+Currently, the following checks will suppress some violations that should not be suppressed when using `supportContextStrategyChecks`, 
```

Comment:
Put period `.` instead of `,` at the sentence end.

Reply:
done